std/string.d

```diff
@@ -3092,6 +3236,12 @@ if (isConvertibleToString!Range)
     return strip!(StringTypeOf!Range)(str);
 }
 
+auto strip(Range)(auto ref Range str, string chars)
+if (isConvertibleToString!Range)
+{
+    return strip!(StringTypeOf!Range)(str, chars);
+}
+
```

Comment:
Same goes for these overloads.  Keep the original single-argument overloads grouped together with their respective unittest blocks, followed by the new 2-argument overloads and the new unittests.